presto-main/src/main/java/com/facebook/presto/operator/window/WindowAnnotationsParser.java

```diff
@@ -37,12 +38,15 @@ private WindowAnnotationsParser() {}
     {
         WindowFunctionSignature[] signatures = clazz.getAnnotationsByType(WindowFunctionSignature.class);
         checkArgument(signatures.length > 0, "Class is not annotated with @WindowFunctionSignature: %s", clazz.getName());
+        WindowFunctionOptions[] options = clazz.getAnnotationsByType(WindowFunctionOptions.class);
+        boolean canIgnoreNulls = options.length > 0 && options[0].canIgnoreNulls();
         return Stream.of(signatures)
-                .map(signature -> parse(clazz, signature))
+                .map(signature -> parse(clazz, signature, canIgnoreNulls))
                 .collect(toImmutableList());
     }
 
-    private static SqlWindowFunction parse(Class<? extends WindowFunction> clazz, WindowFunctionSignature window)
+    private static SqlWindowFunction parse(Class<? extends WindowFunction> clazz, WindowFunctionSignature window,
+                                           boolean canIgnoreNulls)
```

Comment:
See https://github.com/airlift/codestyle

We either wrap all arguments or none. If wrapped, they should use a continuation indent of 8 spaces (not aligned to parentheses).